openfl/interface/interactive_api/experiment.py

```diff
@@ -0,0 +1,335 @@
+# We need assigner interface
+import os
+from copy import deepcopy
+from logging import getLogger
+import functools
+from collections import defaultdict
+from openfl.federated import Plan
+from pathlib import Path
+from importlib import resources
```

Comment:
This breaks compatibility with python < 3.7. It would be good to investigate whether there's an alternative to this

Reply:
Done